ansible/roles/master/tasks/main.yml

```diff
@@ -70,6 +70,14 @@
 - name: write the kubecfg (auth) file for kubectl
   template: src=kubectl.kubeconfig.j2 dest={{ kube_config_dir }}/kubectl.kubeconfig
 
+- name: write the config files for kubelet
+  template: src=kubelet.j2 dest={{ kube_config_dir }}/kubelet
```

Comment:
I'd rather 'hackily' use a src=../../node/templates/kubelet.j2 rather than duplicate the template....